rbi/core/nil_class.rbi

```diff
@@ -16,7 +16,13 @@ class NilClass < Object
   # [`Object`](https://docs.ruby-lang.org/en/2.7.0/Object.html), effectively the
   # same as calling `#==`, but typically overridden by descendants to provide
   # meaningful semantics in `case` statements.
-  def ===(_); end
+  sig do
+    params(
+        arg0: BasicObject,
```

Comment:
would it make sense to widen the argument's type to `T.anything`?

Reply:
+1